userspace/libscap/CMakeLists.txt

```diff
@@ -127,10 +127,6 @@ add_library(scap_event_schema
 		../../driver/flags_table.c
 		../../driver/tp_table.c)
 
-if(USE_BUNDLED_ZLIB AND NOT MINIMAL_BUILD)
```

Comment:
Needed to avoid:
```
In file included from /libs/userspace/libscap/scap_savefile_api.h:24,
from /libs/userspace/libscap/scap.h:21,
from /libs/userspace/libscap/scap_engine_util.c:20:
/libs/userspace/libscap/scap_zlib.h:23:10: fatal error: zlib.h: No such file or directory
23 | #include <zlib.h>
|          ^~~~~~~~
compilation terminated.
make[3]: *** [libscap/CMakeFiles/scap_engine_util.dir/build.make:76: libscap/CMakeFiles/scap_engine_util.dir/scap_engine_util.c.o] Error 1
make[3]: *** Waiting for unfinished jobs....
In file included from /libs/userspace/libscap/scap_savefile_api.h:24,
from /libs/userspace/libscap/scap.h:21,
from /libs/userspace/libscap/engine/noop/noop.c:29:
/libs/userspace/libscap/scap_zlib.h:23:10: fatal error: zlib.h: No such file or directory
23 | #include <zlib.h>
|          ^~~~~~~~
compilation terminated.
make[3]: *** [libscap/engine/noop/CMakeFiles/scap_engine_noop.dir/build.make:76: libscap/engine/noop/CMakeFiles/scap_engine_noop.dir/noop.c.o] Error 1
make[2]: *** [CMakeFiles/Makefile2:1322: libscap/engine/noop/CMakeFiles/scap_engine_noop.dir/all] Error 2
make[2]: *** Waiting for unfinished jobs....
In file included from /libs/userspace/libscap/scap_savefile_api.h:24,
from /libs/userspace/libscap/scap.h:21,
from /libs/userspace/libscap/ringbuffer/ringbuffer.c:23:
/libs/userspace/libscap/scap_zlib.h:23:10: fatal error: zlib.h: No such file or directory
23 | #include <zlib.h>
|          ^~~~~~~~
compilation terminated.
make[3]: *** [libscap/CMakeFiles/scap_engine_util.dir/build.make:104: libscap/CMakeFiles/scap_engine_util.dir/ringbuffer/ringbuffer.c.o] Error 1
In file included from /libs/userspace/libscap/ringbuffer/../scap_savefile_api.h:24,
from /libs/userspace/libscap/ringbuffer/../scap.h:21,
from /libs/userspace/libscap/ringbuffer/devset.c:8:
/libs/userspace/libscap/ringbuffer/../scap_zlib.h:23:10: fatal error: zlib.h: No such file or directory
23 | #include <zlib.h>
|          ^~~~~~~~
compilation terminated.
```
Errors.